packages/databricks-vscode/src/language/notebooks/NotebookInitScriptManager.ts

```diff
@@ -63,11 +139,182 @@ export class NotebookInitScriptManager implements Disposable {
+                            executable,
+                            ["-m", "IPython", file],
+                            {
+                                shell: false,
+                                env: {
+                                    ...((await EnvVarGenerators.getDatabrickseEnvVars(
+                                        this.connectionManager,
+                                        this.pythonExtension,
+                                        this.workspacePath
+                                    )) ?? {}),
+                                    ...((await EnvVarGenerators.getIdeEnvVars()) ??
+                                        {}),
+                                    ...((await this.getUserEnvVars()) ?? {}),
+                                },
+                            }
+                        );
+                        const correctlyFormatttedErrors = stderr
+                            .split(/\r?\n/)
+                            .filter((line) => line.split(":").length > 2);
+                        if (correctlyFormatttedErrors.length > 0) {
+                            someScriptFailed = true;
+                            this.outputWindow.appendLine("=".repeat(30));
+                            this.outputWindow.appendLine(
+                                `Errors in ${path.basename(file)}:`
+                            );
+                            this.outputWindow.appendLine(" ");
+                        }
+                        correctlyFormatttedErrors.forEach((line) => {
+                            const parts = line.split(":");
+                            const [funcName, errorType, ...rest] = parts;
+                            this.outputWindow.appendLine(
+                                `${funcName} - ${errorType}: ${rest.join(":")}`
+                            );
+                        });
+
+                        if (stderr.length > 0) {
+                            ctx?.logger?.error("Notebook Init Script Error", {
+                                stderr: stderr,
+                            });
+                        }
+                    }
+
+                    if (someScriptFailed) {
+                        this.outputWindow.appendLine("\n\n");
+                        window
+                            .showErrorMessage(
+                                "There were errors when running the notebook init script. " +
+                                    "See the Databricks Notebook output window for more details.",
+                                "Open Output Window"
+                            )
+                            .then((choice) => {
+                                if (choice === "Open Output Window") {
+                                    this.outputWindow.show();
+                                }
+                            });
+                        return false;
+                    }
+                    window.showInformationMessage(
```

Comment:
IMHO When everything is fine we should be silent

Reply:
The init scripts take a few seconds to run. That can cause confusion. That's why I also added a progress bar to show that the script is being analysed. This just gives a nice ending to the progress bar.

Users might want to fix any errors (packages and such), and try rerunning the script. No feedback would be weird then. 

Although, the extension does do a bunch of prechecks, so there should not be many missing packages. Which makes this more of an edge case right now and future proofing mechanism. I am ok with removing both notifications (progress bar and success) if the above cases have been considered.